bsdlib/doc/gnss_extension.rst

```diff
@@ -143,11 +143,57 @@ Satellites with elevation angle less than the threshold are automatically exclud
 Use case
    | Socket option:  :c:type:`NRF_SO_GNSS_USE_CASE`
    | Datatype:       :c:type:`uint8_t`
-   | Default         1
-   | Allowed values: 0, 1
+   | Default         :c:type:`NRF_GNSS_USE_CASE_MULTIPLE_HOT_START | NRF_GNSS_USE_CASE_NORMAL_ACCURACY`
+   | Allowed values:
+
+   | :c:type:`NRF_GNSS_USE_CASE_SINGLE_COLD_START`
+   | :c:type:`NRF_GNSS_USE_CASE_MULTIPLE_HOT_START`
+   | :c:type:`NRF_GNSS_USE_CASE_NORMAL_ACCURACY`
+   | :c:type:`NRF_GNSS_USE_CASE_LOW_ACCURACY`
+
+This bitmask is used to specify the combination of the target GNSS performance (single cold or multiple hot start)
+and accuracy (normal or low).
+
+..Note::
+  Normal and Low accuracy is available in MFW 1.2.2 and later
+
+Low Accuracy:
+
+Enabling low accuracy mode will result in the GNSS receiver having a looser acceptance criterion
+for a fix (i.e. error in calculation vs actual position can be larger than Normal Accuracy mode).
+In addition, in the Low Accuracy mode the GNSS receiver may use only 3 satellites to determine a
+fix (in Normal Accuracy mode, it needs 4 or more satellitesi).
+
+For a possible 3 satellite fix to take place, one of the following criteria must be in place first
+and will be valid for a time window of 24 hour:
+
+ | 1) At least 1 fix must have been taken place using 5 or more satellites. You initiate the GPS to start Low accuracy mode, but the radio will not be able to utilize 3 satellite fixing before the criteria above has taken place. In the following time window, any fix that is done using 5 or more satellites will reset the time window and extend the 24-hour period.
+ |
+ | 2) Injecting Altitude (in meter), Altitude uncertainty (0) , and Confidence (100 percent) to the GPS radio using agps data location struct as described below. To be able to stay in Low Accuracy mode this step has to be repeated at least every 24 hour.
+ |
+ | **Caution:** The Altitude in meter has to have an accuracy in the range of +/- 10’s of meter of your actual height. An erroneous altitude will result in a quite sever error in 3 satellite position fix calculation.
+
+.. code-block:: c
 
-The use case parameter determines the target performance.
-A value of 0 indicates single cold start and one indicates multiple hot starts.
+    typedef struct
+    {
+        int32_t latitude          = 0;     // Not known, set to 0
+        int32_t longitude         = 0;     // Not known, set to 0
+        int16_t altitude          = meter; // Your actual height in meter
+        uint8_t unc_semimajor     = 127;   // Both set to 127 when latitude
+        uint8_t unc_semiminor     = 127;   // and longitude are unknown
+        uint8_t orientation_major = 0;     // Not known, set to 0
+        uint8_t unc_altitude      = 0;     // Set to 0
+        uint8_t confidence        = 100;   // Set to 100
+    } nrf_gnss_agps_data_location_t;
```

Comment:
Is it intentional that the typedef and struct definition along with values? Or was the intention to show how to declare a struct to set the altitude?

Reply:
I wrote it like this to describe how to set the values in the struct. Thought this was easy and clear compare to write it as text. Open to suggestion here.

Reply:
@jtguggedal If this struct aproach does make sence - Do you have a suggestion on how we could document which values to set (alternative to use the struct) ?

Reply:
How about this?

```
nrf_gnss_agps_data_location_t location;

location.latitude          = 0;        /* Unknown, set to 0 */
location.longitude         = 0;        /* Unknown, set to 0 */
location.altitude          = altitude; /* Device's actual altitude in meters */
location.unc_semimajor     = 127;      /* Set to 127 when location is unknown */
location.unc_semiminor     = 127;      /* Set to 127 when location is unknown */
location.orientation_major = 0;        /* Unknown, set to 0 */
location.unc_altitude      = 0;        /* Set to 0 */
location.confidence        = 100;      /* Set to 100 */
```

Reply:
Yeah, something like that or alternatively
```
nrf_gnss_agps_data_location_t location = {
        .altitude          = altitude, /* Device's actual altitude in meters */
        .unc_semimajor     = 127,      /* Set to 127 when location is unknown */
        .unc_semiminor     = 127,      /* Set to 127 when location is unknown */
        .confidence        = 100,      /* Set to 100 */
};
```

I don't have strong preference as long as it's valid C

Reply:
@jtguggedal I like that approach. Just wondering if we should also include the values which need to be zero for clarity?

Reply:
We could include them for clarity. In real code, I usual omit those as they're zeroed implicitly anyway, but no harm in being explicit in the docs.

Reply:
The reason I added the whole struct was to also include the filed that are not mention in text. I.e. the one set to zero. I think that is needed